api/src/plugins/auth0.test.ts

```diff
@@ -332,7 +332,7 @@ describe('auth0 plugin', () => {
       expect(res.headers.location).toMatch(HOME_LOCATION);
     });
 
-    test('should redirect to email-sign-up if the user has not acceptedPrivacyTerms', async () => {
+    it('should redirect to returnTo page even if the user has not acceptedPrivacyTerms', async () => {
```

Comment:
```suggestion
    test('should redirect to returnTo page even if the user has not acceptedPrivacyTerms', async () => {
```

This will fix the failing test